env.sh

```diff
@@ -0,0 +1,5 @@
+export DBHOST="unistatsdb.postgres.database.azure.com"
```

Comment:
We shouldn't be publishing this information on a public repository in github

env.sh should be removed, environment variables are explained in the README.md and should suffice for a developer to run the CMS and postgres instance on their local machine